packages/x-charts/package.json

```diff
@@ -47,7 +47,7 @@
     "@react-spring/web": "^9.7.5",
     "clsx": "^2.1.1",
     "prop-types": "^15.8.1",
-    "react-is": "^18.3.1",
+    "react-is": "^19.0.0",
```

Comment:
Would this work with React 18 as well?

Reply:
Good spot. 💯 
We might need to add a `||` condition. 🙈

Reply:
Changed it to `^18.3.1 || ^19.0.0` (same approach as with `@mui/utils`) for flexibility. 😉